### ISSUE TITLE
I can't select a configuration folder

### ISSUE THREAD

**Issue by gpalleschi** [liximomo/vscode-sftp#901]
I can't select a configuration folder. When I select Open Folder nothing happen.

![image](https://user-images.githubusercontent.com/12493269/111180016-e8b55880-85ac-11eb-90dd-09f0cae40c67.png)

My visual studio code version is 1.54.2 and sftp version is 1.12.9.

**Comment by mean-cj** [comment]
A basic configuration file will appear named sftp.json under the .vscode directory. 

project/.vscode/sftp.json

**Comment by glebgolov-ziosem** [comment]
Press `Ctrl+K Ctrl+O`, select a folder or create one, open it in vscode, press `F1`, type and select (pressing `Enter`) `SFTP: Config`.
A basic config file called `sftp.json` will appear under `.vscode` in your project folder.

**Comment by gpalleschi** [comment]
> Press `Ctrl+K Ctrl+O`, select a folder or create one, open it in vscode, press F1, type and select (pressing `Enter`) `SFTP: Config`.
> A basic config file called `sftp.json` will appear under `.vscode` in your project folder.

Thank you @glebgolov-ziosem  I can select my personal folder with ctrl+k ctrl+o. But sftp config button "Open Folder" don't work.

**Comment by glebgolov-ziosem** [comment]
> > Press `Ctrl+K Ctrl+O`, select a folder or create one, open it in vscode, press F1, type and select (pressing `Enter`) `SFTP: Config`.
> > A basic config file called `sftp.json` will appear under `.vscode` in your project folder.
> 
> Thank you @glebgolov-ziosem I can select my personal folder with ctrl+k ctrl+o. But sftp config button "Open Folder" don't work.

I'm sorry, I'm afraid of not being able to follow you. What kind of button are you talking about?

**Comment by gpalleschi** [comment]
Hi @glebgolov-ziosem 
The button i am talking about is this (show the picture), "Open Folder" Button to configure sftp. In the previous release it was ok.
![image](https://user-images.githubusercontent.com/12493269/111615705-58138e00-87e1-11eb-8d72-92c05498b093.png)

**Comment by glebgolov-ziosem** [comment]
Oh right, it doesn't work neither for me. It must be reported as a bug.
I hope at least that you can configure the SFTP for your projects with [the procedure I've explained before](https://github.com/liximomo/vscode-sftp/issues/901#issuecomment-801130406).

**Comment by gpalleschi** [comment]
> Oh right, it doesn't work neither for me. It must be reported as a bug.
> I hope at least that you can configure the SFTP for your projects with [the procedure I've explained before](https://github.com/liximomo/vscode-sftp/issues/901#issuecomment-801130406).

Yes actually I solve the problem with your procedure, thank you very much.
